docs_src/microservices/device/services/device-gpio/GettingStarted.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Device GPIO - Getting Started
+---
+
+# Device GPIO - Getting Started
+
+!!! note
+    Since GPIO `sysfs` interface is **deprecated after Linux version 4.8**, two ABI interfaces are provided: the `sysfs` version and the new `chardev` version. By default, the interface is set to `sysfs`. It can be changed inside `Driver` section of service's configuration. For the `chardev` interface, you need to specify a selected chip, this is also under `Driver` section. See the [Configuration](./Configuration.md) section for more details
```

Comment:
```suggestion
    Since GPIO `sysfs` interface is **deprecated after Linux version 4.8**, two ABI interfaces are provided: the `sysfs` version and the new `chardev` version. By default, the interface is set to `sysfs`. It can be changed inside the `Driver` section of the service's configuration. For the `chardev` interface, you need to specify a selected chip. This is also under `Driver` section. See the [Configuration](./Configuration.md) section for more details
```